meetings/160330_organisers.md

```diff
@@ -1,11 +1,172 @@
+
+- one day on git where we discuss collaborative work (branching, conflict resolution, etc.)
+- 1/2 day on Shell where we could teach the entire Software Carpentry Lesson
+- 1/2 day on licensing (software and data license)
+- 1/2 or 1 day introducing Python or R (could be useful for Data Carpentry follow-up workshops)
+- 1/2 or 1 day Software Carpentry workshop for professors
+
+Vincent also mentionned the interest of ForBio (http://www.forbio.uio.no) to develop a Software Carpentry workshop week. The ambition of Forbio is to train a new generation of biosystematists in the Nordic countries and as many PhDs are coming from abroad, it would be necessary to hold a full week rather than several 1/2 days or 1 day workshops. 
+
+We discussed the possibility to participate more actively in Research school to train younger researchers (PhDs), master Students or even undergraduates. We all agreed there is a need to adapt Software Carpentry workshop "format".
+
+The codata international Training Workshop format was discussed: they offer the possibility to build your own workshop with lessons chosen according to your needs. We all think we could use the same kind of format, with adequate information on the possible prerequisites for each session.
+
+
+#### Actions ####
+Vincent will follow-up with ForBio.
+
+
+## Beyond workshops ##
+
+We already discussed about the possiblity to organize some kind of informal follow-up to our Software & Data Carpentry workshops where attendees can meet up again.
+
+Several ideas were given on what could be done:
+
+- Software Therapy meeting (when attendees start to use what they have learnt and encounter problems: the objective would be to help them and move forward)
+- Short session where a scientist would test the robustness of a developed scientific workflow: all the attendees of this session would try to effectively reproduce his/her work.
+
+After some discussions, we decided to set-up a data therapy subgroup to discuss this matter. This subgroup will be free to decide how, when and what (subjects) to organize. However, it has been clearly noted that to be effective, it is necessary to give an agenda for the couple of months in advance to make sure we attract as many interested persons as possible.
+
+#### Actions ####
+
+Karin will set-up this subgroup and organize a meeting to discuss it. Vincent, Carrie and Anne already expressed their interests.
+
+## git vs github ##
+
+
+Michel asks for more training on github (not only on git) and also mentionned the necessity to train the trainers. 
+
+On the github part, Lex said we could even develop a more github oriented lesson for git where the usage of command line would not be necessary.
+
+About the necessity to train the trainers, we have started to write some documents on how to organize a workshop at UIO or on who can teach what and we think we should develop more UIO-based information.

```

Comment:
Yes, good! The mozilla link is https://www.mozillascience.org/making-friends-with-github.